frappe/utils/redis_wrapper.py

```diff
@@ -27,7 +27,8 @@ def make_key(self, key, user=None, shared=False):
 
 			key = "user:{0}:{1}".format(user, key)
 
-		return "{0}|{1}".format(frappe.conf.db_name, key).encode('utf-8')
+		url = frappe.utils.get_url()
+		return "{0}|{1}|{2}".format(url, frappe.conf.db_name, key).encode('utf-8')
```

Comment:
This isn't a good idea IMO. 

This has to be _optional_ and maybe controllable by web pages if required. `make_key` has 2 options already: user and shared (which is used for site-agnostic cache)

Similarly, it can add another key... which could be URL OR it could be just a generic custom key parameter/generator.